src/gui/accountstate.cpp

```diff
@@ -28,6 +28,10 @@
 #include <QTimer>
 #include <qfontmetrics.h>
 
+namespace {
+static const QLatin1String userSignedOutC("userSignedOut");
```

Comment:
Maybe an inline function to align with the others used?
I'm also wondering whether we should better handle it in httpcredentials along with the user name?

Reply:
I'd say no: independent of how the user authenticated before, the state is signed-out.

That said: if the user re-authenticates, can he/she give in different credentials (e.g. the username), or is the method fixed after authenticating once?